src/services/pocket-relayer.ts

```diff
@@ -235,82 +206,86 @@ export class PocketRelayer {
         });
       }
     }
-    // Exhausted relay attempts; use fallback
+    // Exhausted network relay attempts; use fallback
     if (fallbackAvailable) {
       let relayStart = process.hrtime();
+      let axiosConfig = {};
 
-      const fallbackChoice = new HttpRpcProvider(this.fallbacks[Math.floor(Math.random() * this.fallbacks.length)]);
-      const fallbackPayload : FallbackPayload = {data: rawData.toString(), method: httpMethod, path: relayPath,  headers: null};
-      const fallbackMeta: FallbackMeta = {block_height: 0};
-      const fallbackProof: FallbackProof = {blockchain: blockchain};
-      const fallbackRelay: FallbackRelay = {
-        payload: fallbackPayload,
-        meta: fallbackMeta,
-        proof: fallbackProof,
-      };
-
-      const fallbackResponse = await fallbackChoice.send(
-        '/v1/client/relay',
-        JSON.stringify(fallbackRelay),
-        1200000,
-        false,
-      );
+      // Add relay path to URL
+      const altruistURL = (relayPath === undefined || relayPath === "") ? this.altruists[blockchain] : this.altruists[blockchain] + '/' + relayPath;
```

Comment:
We could probably make this use more modern syntax. Roughly:
```suggestion
      const altruistURL = (relayPath === undefined || relayPath === "") ? this.altruists[blockchain] : `${this.altruists[blockchain]}/${relayPath};
```

Reply:
I'm old and my concatenation style is old.